filters/filter_15_DnsFilter/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "filterId": 15,
-  "name": "AdGuard DNS filter",
+  "name": "(Obsolete) AdGuard DNS filter",
```

Comment:
The list itself is not obsolete so I suggest not changing the name

Reply:
Done.